Misc/stable_abi.toml

```diff
@@ -2406,3 +2406,8 @@
     added = '3.12'
 [const.Py_TPFLAGS_ITEMS_AT_END]
     added = '3.12'
+
+[function._Py_IncRefTotal_DO_NOT_USE_THIS]
+    added = '3.13'
+[function._Py_DecRefTotal_DO_NOT_USE_THIS]
+    added = '3.13'
```

Comment:
3.12?

Reply:
No, they don't exist in the limited API in 3.12. (If you want to fix that, that's up to you :)